website/templates/profile.mako

```diff
@@ -150,7 +149,7 @@
                    "tpl" : "util/render_nodes.mako",
                    "uri" : "/api/v1/profile/${profile["id"]}/public_projects/",
                    "replace" : true,
-                   "kwargs" : {"sortable" : true, "user": ${json.dumps(user)}, "pluralized_node_type": "projects", "skipBindings": true}
+                   "kwargs" : {"sortable" : true, "user": ${ user | sjson, n }, "pluralized_node_type": "projects", "skipBindings": true}
```

Comment:
I'm still not crazy about calling this `sjson` in templates. `to_json` would be my preference, followed by `j`.

I can live with it as-is, though.